flow-plugins/vaadin-gradle-plugin/src/functionalTest/kotlin/com/vaadin/gradle/AbstractGradleTest.kt

```diff
@@ -0,0 +1,39 @@
+package com.vaadin.gradle
+
+import org.junit.After
+import org.junit.Before
+import java.io.File
+
+/**
+ * Prepares a test Gradle project - creates a temp dir for the [testProject] and allow you to run gradle
+ * tasks. See [TestProject] for more details.
+ * @author mavi
+ */
+abstract class AbstractGradleTest {
+
+    // find the newest Vaadin version here: https://github.com/vaadin/platform/tags
+    val vaadinVersion = "20.0.0.alpha3"
```

Comment:
```suggestion
    val flowVersion = System.getenv("vaadin.version").takeUnless { it.isNullOrEmpty() } ?: "7.0-SNAPSHOT"
```
We should probably get the version from the pom in some way.